mindsdb/interfaces/chatbot/chatbot_controller.py

```diff
@@ -74,6 +75,58 @@ def get_chatbot(self, chatbot_name: str, project_name: str = 'mindsdb') -> db.Ch
             'created_at': bot.created_at,
             'is_running': task.active,
             'last_error': task.last_error,
+            'webhook_token': bot.webhook_token,
+        }
+        return bot_obj
+    
+    def get_chatbot_by_id(self, chatbot_id: int) -> db.ChatBots:
```

Comment:
it almost identical with `get_chatbot`
we can reuse the code, call one fnction from another or create third function. 
one of the option:
- `get_chatbot` finds chatbot.id by name and project and then call `get_chatbot_by_id`